crates/gitbutler-testsupport/src/lib.rs

```diff
@@ -60,40 +60,40 @@ pub fn init_opts_bare() -> git2::RepositoryInitOptions {
     opts
 }
 
-pub mod read_only {
-    use std::{
-        collections::BTreeSet,
-        path::{Path, PathBuf},
-    };
+pub mod writable {
+    use crate::DRIVER;
+    use gitbutler_command_context::CommandContext;
+    use gitbutler_project::{Project, ProjectId};
 
+    pub fn fixture(script_name: &str, project_directory: &str) -> anyhow::Result<CommandContext> {
+        let project = fixture_project(script_name, project_directory)?;
+        CommandContext::open(&project)
+    }
+    pub fn fixture_project(script_name: &str, project_directory: &str) -> anyhow::Result<Project> {
+        let root = gix_testtools::scripted_fixture_writable_with_args(
```

Comment:
This function returns a tempdir, which is not returned and thus dropped.
Retaining it and returning it next to `Project` should do the trick.

Reply:
this solves my issue, thank you!!